Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM python:3.9
+FROM python:3.11
 LABEL maintainer="llindstrom@lco.global"
 
 # the exposed port must match the deployment.yaml containerPort value
 EXPOSE 80
-ENTRYPOINT [ "/usr/local/bin/gunicorn", "tom_demo_base.wsgi", "-b", "0.0.0.0:80", "--access-logfile", "-", "--error-logfile", "-", "-k", "gevent", "--timeout", "300", "--workers", "2"]
+ENTRYPOINT [ "poetry", "run", "gunicorn", "tom_demo_base.wsgi", "-b", "0.0.0.0:80", "--access-logfile", "-", "--error-logfile", "-", "-k", "gevent", "--timeout", "300", "--workers", "2"]
 
 WORKDIR /tom-demo
 
-COPY . /tom-demo
 RUN pip install --upgrade pip && pip install poetry
```

Comment:
We've had headaches not pinning the version of poetry we install. So, can we specify that version 2 (and not 3) to be installed here? (I see you've updated to version 2 in the `pyproject.toml` file).